crates/task-impls/src/transactions.rs

```diff
@@ -144,16 +147,21 @@ impl<
     #[instrument(skip_all, fields(id = self.id, view = *self.cur_view), name = "Transaction Handling Task", level = "error")]
     async fn wait_for_block(
         &self,
-    ) -> Option<(AvailableBlockData<TYPES>, AvailableBlockHeaderInput<TYPES>)> {
+    ) -> Option<(
```

Comment:
This is becoming a bit hard to parse. We should maybe make this into a struct.

Reply:
https://github.com/EspressoSystems/HotShot/pull/2956/commits/8d15da890e00f3ab93edcd880b5fb1160b95768d